src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLVehiclePositionImpl.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.ext.legacygraphqlapi.datafetchers;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import org.opentripplanner.ext.legacygraphqlapi.generated.LegacyGraphQLDataFetchers.LegacyGraphQLVehiclePosition;
+import org.opentripplanner.ext.legacygraphqlapi.generated.LegacyGraphQLTypes.LegacyGraphQLVehicleStopStatus;
+import org.opentripplanner.model.vehicle_position.RealtimeVehiclePosition;
+
+public class LegacyGraphQLVehiclePositionImpl implements LegacyGraphQLVehiclePosition {
+
+
+    @Override
+    public DataFetcher<Double> heading() {
+        return env -> getSource(env).heading;
+    }
+
+    @Override
+    public DataFetcher<String> label() {
+        return env -> getSource(env).label;
+    }
+
+    @Override
+    public DataFetcher<Double> lat() {
+        return env -> getSource(env).lat;
+    }
+
+    @Override
+    public DataFetcher<Double> lon() {
+        return env -> getSource(env).lon;
+    }
+
+    @Override
+    public DataFetcher<Object> nextStop() {
```

Comment:
Isn't there any type for next stop info/status?

Reply:
This type is automatically generated from the a configuration file: https://github.com/ibi-group/OpenTripPlanner/blob/2d1750622e48a41d033b89c5d2ce68695503401c/src/ext/java/org/opentripplanner/ext/legacygraphqlapi/generated/graphql-codegen.yml#L69

But you're right, we should actually use a better type here. 

@hannesj @optionsome Could we introduce a supertype for `Station` and `Stop` (in a separate PR)?